modules/deployments-ab-testing-lb.adoc

```diff
@@ -214,18 +214,19 @@ The application is deployed and a service is created. This is the first shard.
 +
 [source,terminal]
 ----
+$ oc expose deploymentconfig ab-example-a --name=ab-example --selector=ab-example\=true
 $ oc expose svc/ab-example-a --name=ab-example
 ----
 
-. Browse to the application at `ab-example-<project>.<router_domain>` to verify you see the `v1` image.
+. Browse to the application at `ab-example-<project_name>.<router_domain>` to verify you see the `v1` image.
 
 . Create a second shard based on the same source image and label as the first shard, but with a different tagged version and unique environment variables:
 +
 [source,terminal]
 ----
 $ oc new-app openshift/deployment-example:v2 \
     --name=ab-example-b --labels=ab-example=true \
-    SUBTITLE="shard B" COLOR="red"
+    SUBTITLE="shard B" COLOR="red" --as-deployment-config=true
```

Comment:
As same  we never use the default created service ab-example-b . To avoid confusion， we need to delete the default  :
`oc delete svc/ab-example-b`